src/widgets/PaperModal/PaperModalContainer.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  TouchableWithoutFeedback,
+  useWindowDimensions,
+  Modal,
+  KeyboardAvoidingView,
+  View,
+  Pressable,
+} from 'react-native';
+
+import { FlexView } from '../FlexView';
+import { Paper } from '../Paper';
+import { TRANSPARENT_GREY } from '../../globalStyles/index';
+
+export const PaperModalContainer = ({ isVisible, onClose, children }) => {
+  const { width, height } = useWindowDimensions();
+
+  return (
+    <Modal
+      visible={isVisible}
+      animationType="fades"
+      hardwareAccelerated
+      onRequestClose={onClose}
+      transparent
+    >
+      <Pressable style={{ flex: 1 }} onPress={onClose}>
+        <FlexView
+          flex={1}
+          justifyContent="center"
+          alignItems="center"
+          style={{ backgroundColor: TRANSPARENT_GREY }}
+        >
+          <Paper paddingHorizontal={0}>
+            <View style={{ height: height / 2, width: width / 2 }}>
+              <TouchableWithoutFeedback
+                onPress={() => {
+                  /* This Touchable is capturing gestures such that only the transparent overlay
```

Comment:
Hmm this is just a theory, but for me, when trying to download from a sensor low enough in the page (below the midpoint...ish?), touching the textbox is dismissing the modal as well.

Reply:
@katherine-sussol 

Interesting - I can't really recreate this at all but it's possible that I don't understand.

I made this GIF which was going to be a really good idea but you can't see me clicking, so it's kind of useless. But I went to the effort to make it, so you get it anyway!!!

![hello1](https://user-images.githubusercontent.com/35858975/106978298-df241e00-67c0-11eb-8634-18fad24eabed.gif)

This is me clicking anywhere on the white modal area (it doesn't close) from a modal opened from the bottom sensor...

🤷

Reply:
Haha I like the GIF idea, thanks for sharing, shame about the clicking! That's weiiiird I'm gonna try cleaning up the DB and setting up some new sensors from scratch to make sure I'm not going crazy. It only happens when I focus a text input (e.g. Email or Comment).

Reply:
Oh - I thought you meant to white area.

If it's to do with the text input, it could be to do with your resolution - maybe the text input is in such a position where it gets fully covered with the keyboard and the keyboard has to close, i've seen that before - but not causing the whole modal to close also 😕